doc/Command-Reference.md

```diff
@@ -4108,6 +4108,37 @@ This command is used to delete the syslog server configured.
   Restarting rsyslog-config service...
   admin@str-s6000-acs-11:~$
   ```
+
+# DHCP Relay Destination IP address Configuration Commands 
+
+This sub-section of commands is used to add or remove the DHCP Relay Destination IP address on a VLAN interface.
+
+**config vlan dhcp_relay add** 
+
+This command is used to add a DHCP Relay Destination IP address to the a VLAN.  Note that more that one DHCP Relay Destination IP address can be added on a VLAN interface.
+
+- Usage: config vlan dhcp_relay add <vlan-id> <dhcp_relay_destination_ip>
+- Example: 
+  ```
+  admin@str-s6000-acs-11:~$ sudo config vlan dhcp_relay add 1000 7.7.7.7
+  Added DHCP relay destination address 7.7.7.7 to Vlan1000
+  Restarting DHCP relay service...
+  Running command: systemctl restart dhcp_relay
+  admin@str-s6000-acs-11:~$ 
+  ```
+
+**config vlan dhcp_relay delete**
+
+This command is used to delete the DHCP Relay Destination IP address configured on a VLAN interface. 
```

Comment:
Change to: `This command is used to delete a configured DHCP Relay Destination IP address from a VLAN interface.`